pkg/controllers/node/health/controller.go

```diff
@@ -113,7 +145,7 @@ func (c *Controller) Reconcile(ctx context.Context, node *corev1.Node) (reconcil
 		}
 		if !clusterHealthy {
 			c.recorder.Publish(NodeRepairBlockedUnmanagedNodeClaim(node, nodeClaim, fmt.Sprintf("more then %s nodes are unhealthy in the cluster", allowedUnhealthyPercent.String()))...)
-			return reconcile.Result{}, nil
+			return reconcile.Result{RequeueAfter: 5 * time.Minute}, nil
```

Comment:
Just checking: This is more of a fix than a perf benefit, right? We probably should have been requeueing before but we weren't and that meant that the cluster could have gone healthy after being unhealthy and we wouldn't have requeued a node again until after we got a heartbeat?